platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileListCommand.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.application.service.command;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import org.apache.karaf.bundle.core.BundleService;
+import org.apache.karaf.features.Feature;
+import org.apache.karaf.features.FeaturesService;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.admin.application.service.ApplicationService;
+import org.codice.ddf.admin.application.service.ApplicationServiceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Command(scope = "admin", name = "profile-list", description = "Lists available install profiles")
+@Service
+public class ProfileListCommand extends AbstractProfileCommand {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProfileInstallCommand.class);
+
+    @Override
+    protected void doExecute(ApplicationService applicationService,
+                   FeaturesService featuresService, BundleService bundleService) {
+        listProfiles(applicationService);
+
+        if (showDevProfiles && developmentProfilePath.toFile().exists()) {
+            listDevProfiles();
+        }
+    }
+
+    private void listProfiles(ApplicationService applicationService) {
+        printSectionHeading("Available Profiles:");
+        applicationService.getInstallationProfiles().stream()
```

Comment:
I don't think that will be necessary based on the way we are using profiles. the only use of the `install=` attribute in the features files that we use to create those profiles is on the top level profile feature itself. see: https://github.com/codice/ddf/blob/master/distribution/install-profiles/src/main/resources/features.xml